raiden/ui/web/src/app/components/token-network/token-network.component.ts

```diff
@@ -47,6 +48,10 @@ export class TokenNetworkComponent implements OnInit, OnDestroy {
 
     public refreshing = true;
 
+    public get main(): boolean {
```

Comment:
This getter should be in `RaidenService`, and gotten from there, service is the one that knows better the config, no need to make other components tied to these internal config details